media/css/firefox/developer-quantum-whatsnew.scss

```diff
@@ -0,0 +1,181 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import
+    '../pebbles/includes/lib';
+
+$base-blue: #306ffd;
+
+@mixin section-curve {
+    position: absolute;
+    display: block;
+    left: 0;
+    content: '';
+    background-position: center bottom;
+    background-repeat: no-repeat;
+    width: 100%;
+    visibility: visible;
+    z-index: 10;
+}
+
+//*------------------------------------------------------------------*/
+// Intro
+.section-intro {
+    background-color: #fff;
+    background-position: center bottom;
+    background-repeat: no-repeat;
+    text-align: center;
+
+    @media #{$mq-tablet} {
+        @include background-size(1440px 1034px);
+        background-image: url('/media/img/firefox/products/developer/quantum/whatsnew-hero-background.png');
+        background-position: 0 0;
+        padding-bottom: 40px;
+
+        &:after {
+            @include background-size(3000px 62px);
+            @include section-curve;
+            background-image: url('/media/img/firefox/products/developer/quantum/curve-download-bottom-whatsnew.svg');
+            height: 62px;
+            bottom: -1px;
+        }
+    }
+
+    @media #{$mq-desktop} {
+        @include at2x('/media/img/firefox/products/developer/quantum/whatsnew-hero-background.png', 1780px, 1278px);
+        background-position: 0 -169px;
+        padding: 40px 0 60px;
+    }
+
+    .page-title {
+        @include font-size-level2;
+        text-transform: uppercase;
+        margin: 0 auto 20px;
+
+        &:before {
+            display: block;
+            content: '';
+            @include at2x('/media/img/firefox/products/developer/quantum/dev-edition-logo.png', 105px, 108px);
+            margin: 0 auto 20px;
+            width: 105px;
+            height: 108px;
+        }
+    }
+
+    .tagline {
+        max-width: $width-tablet;
+        text-align: center;
+        margin: 0 auto 80px;
+    }
+
+    .features-list {
+        li {
+            margin-bottom: 40px;
+
+            @media #{$mq-tablet} {
+                margin-bottom: 0;
+            }
+        }
+
+        h3 {
+            @include font-size(18px);
```

Comment:
I think you can use the `@include font-size-level5;` mixin here without the other media queries.